Tests/Amplitude+Test.m

```diff
@@ -66,5 +107,20 @@ - (void)flushUploads:(void (^)(void))handler {
     });
 }
 
+- (void)cleanUp {
+    [self cleanUp:kAMPDefaultInstance];
+
+}
+
+- (void)cleanUp:(NSString *)instanceName {
```

Comment:
Clean up the file storage and the NSUserDefault values for the shared instance across the test cases.
We don't need to do this step for SQLite because we save all things in DB and we can get the last row, but in file storage, we need to close the `]}` in order to read the arrays out of the file and we never modify the files.